addons/web/static/src/views/fields/char/char_field.xml

```diff
@@ -10,13 +10,14 @@
                 class="o_input"
                 t-att-class="{
                     'o_field_translate': isTranslatable,
-                    'o_field_placeholder': hasDynamicPlaceholder
+                    'o_field_placeholder': hasDynamicPlaceholder,
+                    'text_muted': !this.props.record.data[props.name],
                 }"
                 t-att-id="props.id"
                 t-att-type="props.isPassword ? 'password' : 'text'"
                 t-att-autocomplete="props.autocomplete or (props.isPassword ? 'new-password' : 'off')"
                 t-att-maxlength="maxLength > 0 and maxLength"
-                t-att-placeholder="props.placeholder"
+                t-att-placeholder="placeholder"
```

Comment:
Change that in every template that overwrite it ? Or let them use props.placeholder since they don't need a dynamic one for the moment ?